src/data-dictionary/events/K8sCronjobSample/K8sCronjobSample.event-definition.md

```diff
@@ -0,0 +1,8 @@
+---
+name: K8sCronjobSample
+type: event
+dataSources:
+  - Kubernetes
+---
+
+This event contains Kubernetes `CronJob` data.
```

Comment:
@htroisi I think we should format all CronJob mentions be in code format (`CronJob`), or all are regular (CronJob) to be consistent, so consider editing this to just CronJob to match the other references.

Reply:
Good point. I'll fix it